regression/benchmark_scripts/configs/example_test_creation.py

```
@@ -0,0 +1,70 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# CONFIDENTIAL, provided under NDA.
+
+import cuopt
```

Flake8 (7.3.0)

[error] 'cuopt' imported but unused

(F401)